src/Modules/Workflows/Models/WorkflowModel.php

```diff
@@ -466,6 +466,32 @@ public function createNew($reuseAutoDraft = true): int
         return $id;
     }
 
+    public function createCopy(): WorkflowModel
```

Comment:
@ojopaul, for classes with an interface, in this case `WorkflowModelInterface`, we should use the interface as the return type instead of using the class directly.

Reply:
Thanks @andergmartins, this has been fixed